docker/oap/docker-entrypoint.sh

```diff
@@ -49,14 +49,24 @@ EOT
 }
 
 generateClusterConsul() {
-    cat <<EOT >> ${var_application_file}
-cluster:
-  consul:
-    serviceName: \${SW_SERVICE_NAME:"SkyWalking_OAP_Cluster"}
-    # Consul cluster nodes, example: 10.0.0.1:8500,10.0.0.2:8500,10.0.0.3:8500
-    hostPort: \${SW_CLUSTER_CONSUL_HOST_PORT:localhost:8500}
-EOT
-}
+     cat <<EOT >> ${var_application_file}
+ cluster:
+   consul:
+     serviceName: \${SW_SERVICE_NAME:"SkyWalking_OAP_Cluster"}
+     # Consul cluster nodes, example: 10.0.0.1:8500,10.0.0.2:8500,10.0.0.3:8500
+     hostPort: \${SW_CLUSTER_CONSUL_HOST_PORT:localhost:8500}
+ EOT
+ }
+
+ generateClusterEtcd() {
+      cat <<EOT >> ${var_application_file}
+  cluster:
+    consul:
```

Comment:
```yaml
cluster:
  etcd:
    serviceName: \${SW_SERVICE_NAME:"SkyWalking_OAP_Cluster"}
    # etcd cluster nodes, example: 10.0.0.1:2379,10.0.0.2:2379,10.0.0.3:2379
    hostPort: \${SW_CLUSTER_ETCD_HOST_PORT:localhost:2379}
```

Reply:
> ```yaml
> cluster:
>   etcd:
>     serviceName: \${SW_SERVICE_NAME:"SkyWalking_OAP_Cluster"}
>     # etcd cluster nodes, example: 10.0.0.1:2379,10.0.0.2:2379,10.0.0.3:2379
>     hostPort: \${SW_CLUSTER_ETCD_HOST_PORT:localhost:2379}
> ```

what is the problem?

Reply:
yeah, `consul `--> `etcd`.

Reply:
Why this be resolved in review status, but actually it isn't.